registries/_format/date-time-local.md

```diff
@@ -0,0 +1,15 @@
+---
+owner: ya7010
+issue: 4759
+description: RFC3339 date-time without the timezone component
+base_type: string
+layout: default
+source_label: TOML
+source: https://toml.io/en/v1.0.0#local-date-time
```

Comment:
This is not TOML-specific, so I wouldn't reference it.

It is `iso-date-time` in [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339#appendix-A) without the optional time-zone.

If you want to be exact about the allowed pattern, you can define it in RFC3339 terms as `full-date "T" partial-time`.